Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -1909,17 +1909,24 @@ extension BrowserViewController: TabManagerDelegate {
                 make.left.right.top.bottom.equalTo(self.webViewContainer)
             }
             
-            webViewContainerBackdrop.alpha = 0
+            webView.isHidden = true
+            webViewContainerBackdrop.alpha = 1
+            webViewContainerBackdrop.backgroundColor = UIColor.theme.ecosia.primaryBackground
             navigationController?.view.backgroundColor = UIColor.theme.browser.background
             
             let animation = CABasicAnimation(keyPath: "transform")
             animation.duration = 0.35
             animation.fromValue = CATransform3DMakeScale(0.6, 0.6, 1)
             animation.timingFunction = .init(name: .easeInEaseOut)
-            
-            webView.layer.add(animation, forKey: "scale")
             firefoxHomeViewController?.view.layer.add(animation, forKey: "scale")
 
+            DispatchQueue
+                .main
+                .asyncAfter(deadline: .now() + 0.35) { [weak webView, weak webViewContainerBackdrop] in
+                    webView?.isHidden = false
```

Comment:
Hacky solution but we don't need the webview visible while animating.

Reply:
ok I see.. animations with webviews involved are tricky